admin/modules/tools/spamlog.php

```diff
@@ -161,12 +161,11 @@
 		$order = "desc";
 	}
 
-	// Pagination stuff
-	$sql           = "
-		SELECT COUNT(sid) as count
+	$query = $db->query("
+		SELECT COUNT(sid) AS count
 		FROM ".TABLE_PREFIX."spamlog
```

Comment:
It should be `$db->simple_select()` actually since there are no `JOIN`s. Same applies to the 2nd query. And `l.` is not needed anywhere.